paradise_papers_django/paradise_papers_django/settings/production.py

```diff
@@ -0,0 +1,14 @@
+from .base import *
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+#Connect to Neo4j Database
+config.DATABASE_URL = 'bolt://paradisepapers:paradisepapers@174.138.63.95:7687'  # default
+
+
+MIDDLEWARE += ['whitenoise.middleware.WhiteNoiseMiddleware', ]
+
+STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
+
+ALLOWED_HOSTS = []
```

Comment:
Need to find a way to add the host

Reply:
in allow_host we need to add : ` 'paradise-papers-django.herokuapp.com'` so it can work on heroku